src/govuk/components/accordion/accordion.test.js

```diff
@@ -329,7 +355,41 @@ describe('/components/accordion', () => {
 
           const firstSectionToggleText = await page.evaluate(() => document.body.querySelector('.govuk-accordion__section-toggle-text').innerHTML)
 
-          expect(firstSectionToggleText).toBe('Dangos<span class="govuk-visually-hidden"> adran</span>')
+          expect(firstSectionToggleText).toBe('Dangos')
+        })
+
+        it('should localise "Show section" aria-label based on data attribute', async () => {
+          await goToComponent(page, 'accordion', {
+            exampleName: 'with-translations'
+          })
+
+          const showSectionDataAttribute = await page.evaluate(() =>
+            document.body
+              .querySelector('.govuk-accordion')
+              .getAttribute('data-i18n.show-section-aria-label')
```

Comment:
We alright using the [`getAttribute()`](https://github.com/alphagov/govuk-frontend/blob/main/lib/puppeteer-helpers.js#L121) helper?

Reply:
Actually, probably more a tidy up PR than a tidy up commit.  Most tests in the file use that pattern for reading attributes, so that'll muddy the changes quite a bit.

To be fair, the `getAttribute` helper may not make things better in its current shape as it'll involve 2 interractions with the browser:

1. find the element
2. get its attribute

instead of a single call.

I'd be keen to rework `getAttribute` into something like that to avoid the query the element than get steps.

```js
function getAttribute(pageOrElement, selectorOrAttributeName, attributeName) {
	if (pageOrElement instance of ElementHandle) {
		return pageOrElement.evaluate((el, name) => el.getAttribute('name'), selectorOrAttributeName)
	}
	
	return pageOrElement.$eval(selectorOrAttributeName, (el, name) => el.getAttribute('name'), attributeName)
}
```

Reply:
Or switch to Cypress where it does all this for us 🙃